packages/synthetix-main/contracts/modules/core/VaultModule.sol

```diff
@@ -167,7 +169,7 @@ contract VaultModule is IVaultModule {
     }
 
     /**
-     * @dev See {IVaultModule-getPositionDebt}.
+     * @dev Returns the debt for the given account, pool, and collateral type.
```

Comment:
should clarify that this function will also trigger a debt distribution chain update, which is why this is a writable function

Reply:
possible include reccomendation to use `callStatic` if in ethers (is the function users are looking for)

Reply:
Implemented